src/fidesops/db/base_class.py

```diff
@@ -23,16 +24,16 @@
 
 def get_key_from_data(data: Dict[str, Any], cls_name: str) -> str:
     """
-    Extracts key from data and slugifies. If no key, uses a slugified name.
+    Extracts key from data, validates. If no key, uses a slugified name.
```

Comment:
I use slugify specifically to mean hyphens as separators which is no longer true - I'd do a find/replace for every instance of slugify/slugified in our code base, including renaming the slugify function.